src/main/java/org/openstreetmap/atlas/checks/validation/relations/RouteRelationCheck.java

```diff
@@ -0,0 +1,845 @@
+     * @return a boolean indicating whether the stop or platform is too far from the track
+     */
+
+    private boolean stopPlatformTooFarFromTrackCheck(final List<Location> signLocations,
+            final List<PolyLine> allEdgePolyLines)
+    {
+        final Distance threshHold = Distance.meters(15.0);
+        SnappedLocation minSnap = null;
+        // Location minLoc = null
+
+        for (final Location location : signLocations)
+        {
+            for (final PolyLine edges : allEdgePolyLines)
+            {
+                final SnappedLocation snappedTo = location.snapTo(edges);
+                if (minSnap == null || snappedTo.compareTo(minSnap) < 0)
+                {
+                    minSnap = snappedTo;
+                }
+            }
+        }
+
+        return minSnap != null && minSnap.getDistance().isGreaterThan(threshHold);
+    }
+
+    /**
+     * @param routeRelation
+     *            the public transport route relation. this method checks whether the route relation
+     *            is contained in a route master
+     * @return an instance of CheckRouteMasterValues containing information about whether this
+     *         public transport route is contained in a route master
+     */
+    private boolean testContainedInRouteMasters(final Relation routeRelation)
+    {
+        final Iterable<Relation> relationsInAtlas = routeRelation.getAtlas().relations();
+        final Spliterator<Relation> spliterator = relationsInAtlas.spliterator();
+
+        return StreamSupport.stream(spliterator, false)
+                .filter(relation -> Validators.isOfType(relation, RelationTypeTag.class,
+                        RelationTypeTag.ROUTE_MASTER))
+                .flatMap(relation -> relation.members().stream().map(RelationMember::getEntity))
+                .filter(member -> member.getType().equals(ItemType.RELATION))
+                .filter(member -> Validators.isOfType(member, RelationTypeTag.class,
+                        RelationTypeTag.ROUTE))
+                .anyMatch(member -> Long.toString(member.getIdentifier())
+                        .equals(Long.toString(routeRelation.getIdentifier())));
+    }
+
+    /**
+     * Return the list of instructions that describes inconsistency of any tags in the group of
+     * network, operator, ref, and colour between a route master and its member routes
+     *
+     * @param rel
+     *            The route master relation under check
+     * @return the list of instructions that describes inconsistency
+     */
+    private List<String> testNetworkOperatorRefColourTag(final Relation rel)
+    {
+        final List<String> instructionsAdd = new ArrayList<>();
+        final Optional<String> networkTag = rel.getTag("network");
+        final Optional<String> operatorTag = rel.getTag("operator");
+        final Optional<String> refTag = rel.getTag("ref");
+        final Optional<String> colourTag = rel.getTag("colour");
```

Comment:
It would be great to add operator and colour as tag enums in Atlas.
```suggestion
        final Optional<String> networkTag = rel.getTag(NetworkTag.KEY);
        final Optional<String> operatorTag = rel.getTag("operator");
        final Optional<String> refTag = rel.getTag(ReferenceTag.KEY);
        final Optional<String> colourTag = rel.getTag("colour");
```